exporter/rabbitmqexporter/integration_test.go

```diff
@@ -0,0 +1,162 @@
+// Copyright The OpenTelemetry Authors
+// SPDX-License-Identifier: Apache-2.0
+
+package rabbitmqexporter
```

Comment:
```suggestion
//go:build integration

package rabbitmqexporter
```

If you want this to be treated as an integration test, you'll want to add a build tag to show this. Another example is [here](https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/cf2afd0db91d01e001804efedbb789e6015a4a8b/receiver/dockerstatsreceiver/integration_test.go#L4). This will make it only run when we're specifically running integration tests, and not with unit tests. This build tag gets picked up [here in the makefile](https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/cf2afd0db91d01e001804efedbb789e6015a4a8b/Makefile.Common#L32), for context.